styles/mixins/button.js

```diff
@@ -1,11 +1,15 @@
 module.exports = function () {
   return function (mixin, bgColor, color) {
-    return {
-      'background-color': bgColor,
-      color,
-      display: 'inline-block',
-      'line-height': 1,
-      'text-align': 'center'
+    const activeBgColor = 'red';
```

Comment:
If we add support for those functions,we could still do darker and lighter in yaml